CodeEdit/Features/Welcome/Views/RecentProjectsListView.swift

```diff
@@ -118,4 +120,23 @@ struct RecentProjectsListView: View {
         let projectsURL = recentProjectPaths.map { URL(filePath: $0) }
         recentProjects = projectsURL
     }
+    public func donateSearchableItems() {
+        let searchableItems = recentProjects.map { entity in
+            let attributeSet = CSSearchableItemAttributeSet(contentType: .content)
+            attributeSet.title = entity.lastPathComponent
+            attributeSet.relatedUniqueIdentifier = entity.path()
+            return CSSearchableItem(
+                uniqueIdentifier: entity.path(),
+                domainIdentifier: "app.codeedit.CodeEdit.ProjectItem",
+                attributeSet: attributeSet
+            )
+        }
+        CSSearchableIndex.default().indexSearchableItems(searchableItems) { error in
+            if let error = error {
+                print(error)
+            } else {
```

Comment:
No need for a success message. The message for the error will eventually be integrated into a better logging solution for the application.